examples/s4u/README.doc

```diff
@@ -59,6 +59,11 @@ documentation, but it should remain readable directly.
    until one activity of the set completes, no matter which terminates
    first.   
 
+ - <b>Block and Release semaphore object</b>.
+   @ref examples/s4u/s4u-synchro-semaphore/s4u-synchro-semaphore.cpp\n
+   The @ref simcall_sem_acquire and @ref simcall_sem_release functions
```

Comment:
We surely don't want the user to call the simcall_ref_acquire function directly.